templates/forms/premium-versions-upgrade-handler.php

```diff
@@ -20,20 +20,42 @@
     $plugin_name     = $plugin_data['Name'];
     $plugin_basename = $fs->get_plugin_basename();
 
+    $license = $fs->_get_license();
+
+    if ( ! is_object( $license ) ) {
+        $purchase_url = $fs->pricing_url();
+    } else {
+        $subscription = $fs->_get_subscription( $license->id );
+
+        $purchase_url = $fs->checkout_url(
+            is_object( $subscription ) ?
+                ( 1 == $subscription->billing_cycle ? WP_FS__PERIOD_MONTHLY : WP_FS__PERIOD_ANNUALLY ) :
+                WP_FS__PERIOD_LIFETIME,
+            false,
+            array( 'licenses' => $license->quota )
+        );
+    }
+
     $message = sprintf(
         fs_text_inline( 'There is a new version of %s available.', 'new-version-available-message', $slug ) .
-        fs_text_inline( ' %sRenew your license now%s to access version %s security & feature updates, and support.', 'renew-license-now', $slug ),
+        fs_text_inline( ' %s to access version %s security & feature updates, and support.', 'renew-license-now', $slug ),
```

Comment:
@fajardoleo `'renew-license-now'` is now conflicting since it's also used as the string key here:
```
fs_text_inline( 'Renew your license now', 'renew-license-now', $slug )
```

I suggest renaming the key to `'x-for-updates-and-support'`.